scripts/rs/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
```

Comment:
should we update the folder to procdockerstatsd to be more explicit?

if we like a single crate to have multiple bin, then updating the main.rs into procdockerstatsd.rs will be better for future.